app/views/admin/collections/show.html.erb

```diff
@@ -30,21 +50,26 @@ Unless required by applicable law or agreed to in writing, software distributed
     </div>
     <div class="col-md-3">
       <p><strong>Poster:</strong></p>
-      <p><%= @collection.poster.original_name if @collection.poster.present? %></p>
-      <% if @collection.poster.present? %>
+      <p id="poster_original_name"><%= @collection.poster.original_name if @collection.poster.present? %></p>
       <div class="well-vertical-spacer">
-        <%= image_tag(poster_admin_collection_path(@collection), class: 'img-responsive') %>
+        <% if @collection.poster.present? %>
+          <%= image_tag(poster_admin_collection_path(@collection), class: 'img-responsive', id: 'poster-image') %>
+        <% else %>
+          <img src="" class="img-responsive" id="poster-image">
+        <% end %>
       </div>
-      <% end %>
       <div>
         <%= form_for @collection, url: attach_poster_admin_collection_path(@collection.id), html: {method: "post"} do |form| %>
-        <%= form.file_field :poster, class: "filedata", style: "height:0px;width:0px;" %>
-        <% if @collection.poster.present? %>
-        <input type="submit" class="btn btn-danger btn-xs" value="Remove Poster" />
-        <% else %>
+        <%= form.file_field :poster, id: 'poster_input', class: "filedata", style: "height:0px;width:0px;" %>
         <input type="button" class="btn btn-primary btn-xs" onclick="$('.filedata').click();" value="Upload Poster" />
+        <input type="submit" class="btn btn-danger btn-xs" value="Remove Poster" />
         <% end %>
-        <% end %>
+      </div>
+      <div class="well-vertical-spacer">
+        <div id='cropper-alert' class="alert" role="alert" style="display: none; padding: 5px; margin-top: 5px"></div>
```

Comment:
I'd give this element an extra class noting what type of alert function its providing.  `alert-info`, `alert-warning`, `alert-danger`, etc...  https://getbootstrap.com/docs/3.3/components/#alerts

Reply:
The cropper js adds the appropriate alert type class depending on error state it generates..

Reply:
Ah, nice.